src/compat/posix/include/unistd.h

```diff
@@ -17,6 +17,8 @@
 #include <sys/stat.h>
 #include <sys/cdefs.h>
 
+#include <sys/uio.h>
```

Comment:
Probably, I misunderstood something, but this file is already in embox repository:
https://github.com/embox/embox/blob/master/src/compat/posix/include/sys/uio.h